bazel/src/main/java/com/google/api/codegen/bazel/Buildozer.java

```diff
@@ -82,6 +82,9 @@ public String getAttribute(Path bazelBuildFile, String target, String attribute)
       if (value.equals("(missing)")) {
         return null;
       }
+      if (value.charAt(0) == '"' && value.charAt(value.length() - 1) == '"') {
+        value = value.substring(1, value.length() - 1);
```

Comment:
This does not seem to be related to the spaces issue. How did this thing work before and why is it needed?
Minor: another lazy option of doign it coudl be `value.startsWith("\"") && value.endsWith("\"")`, though the currenv version is more "performant" as it works directly with characters (pretending like it matters here...).

Reply:
It is related. I will add a comment :) The thing is:

```
fenster@fenster ws:~/googleapis$ buildozer 'print ruby_cloud_title' //google/api/servicecontrol/v1:servicecontrol_ruby_gapic
"Service Control API V1"
fenster@fenster ws:~/googleapis$ buildozer 'print name' //google/api/servicecontrol/v1:servicecontrol_ruby_gapic
servicecontrol_ruby_gapic
```

So it adds quotes if the string has spaces inside. We don't need those quotes, so I'm stripping them.

Reply:
Added comment